src/components/Search/Button/EditRecord.js

```diff
@@ -1,25 +1,24 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 // @flow
 import * as React from 'react';
-import { Button, PaneMenu, Icon } from '@folio/stripes/components';
+import { Button, Icon } from '@folio/stripes/components';
 import { connect } from 'react-redux';
-import { uuid } from '../../../shared/utils/Function';
-import { Localize, findParam, buildUrl } from '../../../shared';
-import { lockRecordAction } from '../../Cataloguing/Actions';
+import { Localize, findParam } from '../../../shared';
+// import { lockRecordAction } from '../../Cataloguing/Actions';
 
 const EditRecordButton = ({ ...props }) => {
-  const lockRecord = id => {
-    const { dispatch, store } = props;
-    const user = store.getState().okapi.currentUser.username;
-    const idLock = store.getState().okapi.currentUser.id;
-    console.log(idLock);
-    const payload = {
-      user,
-      id,
-      idLock,
-    };
-    dispatch(lockRecordAction(payload));
-  };
+  // const lockRecord = id => {
+  //   const { dispatch, store } = props;
+  //   const user = store.getState().okapi.currentUser.username;
+  //   const idLock = store.getState().okapi.currentUser.id;
+  //   console.log(idLock);
+  //   const payload = {
+  //     user,
+  //     id,
+  //     idLock,
+  //   };
+  //   dispatch(lockRecordAction(payload));
+  // };
   const handleClickEdit = () => {
     const { router, toggleFilterPane } = props;
     const id = findParam('id');
```

Comment:
Ah, now I see why you uncommented `lockRecord`: Lint won't let you keep the function around if it's not called anywhere. 👌